main.py

```diff
@@ -1,10 +1,62 @@
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main():
```

Comment:
The logic of this function is better divided into several functions.
For example, a separate function for creating a guild/race.